controllers/childrenController.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../models");
```

Comment:
Why are you guys switching between import and require. Pick one and stick with it. It shouldn't be different between your front-end and back-end.